src/components/text-input/daikin-text-input.ts

```diff
@@ -179,10 +220,19 @@ export class DaikinTextInput extends LitElement {
         @change=${(e: Event) => this.dispatchEvent(new Event("change", e))}
         @input=${this._handleInput}
       />
-      <slot name="right-icon">
-        <span class="block -mr-1"></span>
-      </slot>
-    </div>`;
+      <div class=${cvaIcon({ icon: "left", disabled: this.disabled })}>
+        <slot
+          name="left-icon"
+          @slotchange=${(event: Event) => this._handleSlotChange(event, "left")}
```

Comment:
Actually we can simplify this as following since we can obtain slot name (`left-icon` or `right-icon`) with `(event.target as HTMLSlotElement).name`.

```suggestion
          @slotchange=${this._handleSlotChange}
```